static/stylesheets/application.css

```
@@ -1040,10 +1040,9 @@ h6 + .id-link-button {
 .article ol>li {
     padding: 5px 0 5px 10px;
     margin: 0 0 15px 30px;
-    border-left: solid 1px #e6e6e6;
 }
 .article ol>li::marker {
-    color: #888;
+    color: #000000;
     font-size: 1.6rem;
```

手順の数字の大きさが26pxくらいなのでかなり大きいなと思いました。
黒ではっきりした明暗になったので22pxくらいで良いのかも？
```suggestion
    font-size: 1.4rem;
```